File: Assignment 3/program1.c
#include<stdio.h>

void  PrintEven(int iNo)
{
     if(iNo<=0)
    {
        printf("Invalid number");
     }  

    int iCnt=0;
    for(iCnt=1;iCnt<=2*iNo;iCnt++)
    {
        if((iCnt%2)==0)
        {
            printf("%d\n",iCnt);
        }
    }
   

}
int main()
{
    int iValue=0;
    printf("Enter the value: ");
    scanf("%d",&iValue);
    PrintEven(iValue);
    return 0;
}

File: Assignment 26/program1.java
import java.util.*;

class Pattern
{ 
    public void Display(int iNO)
    {
        int iCnt=0;
        char ch='A';

        for(iCnt=1;iCnt<=iNO;iCnt++)
        {
            System.out.print(ch+"\t");
            ch++;
        }

    }

}

class program1
{
    public static void main(String A[])
    { 
        int iValue=0;

        Scanner sobj=new Scanner(System.in);
        System.out.println("Enetr Number N:");
        iValue=sobj.nextInt();

        Pattern pobj=new Pattern();
        pobj.Display(iValue);

    }
}